prg_test_helpers.go

```diff
@@ -34,32 +32,30 @@ func GetNextUInt64NewPRGWithSalt(
 	seed []byte,
 	salt []byte,
 ) (uint64, error) {
+	t.Helper()
 
-	randResult := o.Script(
+	var value uint64
+	err := o.Script(
 		"xorshift128plus/next_uint64",
 		WithArg("sourceOfRandomness", seed),
 		WithArg("salt", salt),
-	)
+	).MarshalAs(value)
 
-	require.NoError(t, randResult.Err)
+	require.NoError(t, err)
 
-	return uint64(randResult.Result.(cadence.UInt64)), randResult.Err
+	// TODO: why do you return err here?
```

Comment:
This is a missed remnant from a previous refactor. The random function passed to `BasicDistributionTest` required this return signature, but I believe we now return just the `uint64`.